Sources/Purchasing/Purchases/PurchaseParams.swift

```diff
@@ -140,6 +159,42 @@ import Foundation
 
         #endif
 
+        #if ENABLE_CUSTOM_ENTITLEMENT_COMPUTATION
+
+        // swiftlint:disable line_length
+        /**
+         * Sets an introductoryOfferEligibility JWS to be included with the purchase. StoreKit 2 only.
+         * - Parameter introductoryOfferEligibilityJWS: The ``introductoryOfferEligibility JWS`` to apply to the purchase.
+         *
+         * Refer to https://developer.apple.com/documentation/storekit/product/purchaseoption/introductoryoffereligibility(compactjws:)
+         * for more information.
+         *
+         * Availability: iOS 15.0+, macOS 15.4+, tvOS 18.4+, watchOS 11.4+, visionOS 2.4+
+         */
+        @available(iOS 15.0, macOS 15.4, tvOS 18.4, watchOS 11.4, visionOS 2.4, *)
+        @objc public func with(introductoryOfferEligibilityJWS: String) -> Self {
+            self.introductoryOfferEligibilityJWS = introductoryOfferEligibilityJWS
+            return self
+        }
+
+        // swiftlint:disable line_length
+        /**
+         * Sets an promotionalOffer JWS to be included with the purchase. StoreKit 2 only.
+         * - Parameter promotionalOfferJWS: The ``promotionalOffer JWS`` to apply to the purchase.
```

Comment:
The parameter name is `promotionalOfferOptions` (and not `promotionalOfferJWS`). 

Also, the documentation of the parameter seems incorrect